conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalytics.java

```diff
@@ -65,13 +60,13 @@
 
     private static final String TAG = "ConvivaAnalytics";
 
-    private Client client;
     private BitmovinPlayer bitmovinPlayer;
     private ContentMetadataBuilder contentMetadataBuilder = new ContentMetadataBuilder();
     private ConvivaConfiguration config;
-    private int sessionId = Client.NO_SESSION_KEY;
-    private PlayerStateManager playerStateManager;
+    private Context context;
+    private ConvivaVideoAnalytics convivaVideoAnalytics;
     private MetadataOverrides metadataOverrides;
+    private boolean activeSession = false;
 
```

Comment:
Just to double-check: the Ad tracking works differently than on iOS? No need for an `adAnalytics` object? 🤔

Reply:
Correct. There's an option to add more ad info but this matches what the previous integration was doing. Here's a snapshot of touchstone with this PR implemented:
![Snip20210614_152](https://user-images.githubusercontent.com/65041599/121933097-df9a6b80-cd02-11eb-97cd-87d28c91c521.png)

Reply:
👍